pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = 'mmda'
-version = '0.3.8'
+version = '0.3.9'
```

Comment:
```suggestion
version = '0.3.10'
```
0.3.9 is already taken :)

Reply:
@comorado done, can you update your review to accept changes?